### PR TITLE
Add tests to LikeButton.vue

### PR DESCRIPTION
**Pull request by Camc314**
Refactors LikeButton and adds tests

### CODE REVIEW

File: components/Buttons/LikeButton.vue [hunk 1]
```diff
@@ -1,39 +1,33 @@
 <template>
-  <v-btn :dark="dark" icon @click.stop.prevent="toggleFavorite">
-    <v-icon :class="fav ? 'red--text' : ''">
-      {{ fav ? 'mdi-heart' : 'mdi-heart-outline' }}
+  <v-btn icon @click.stop.prevent="toggleFavorite">
```
Comment (ferferga) [inline components/Buttons/LikeButton.vue:2]: Why dark was removed? This makes the button invisible in cards

**Review by ferferga — CHANGES_REQUESTED** [review]

File: components/Buttons/LikeButton.vue (hunk 1 above)
Comment (Camc314) [reply components/Buttons/LikeButton.vue:2]: I'll have another look, but as far as I could see we weren't using it anywhere

Comment (Camc314) [reply components/Buttons/LikeButton.vue:2]: Yeah checked again. The dark prop for like-button isn't used anywhere.

https://github.com/jellyfin/jellyfin-vue/blob/master/components/Buttons/QueueButton.vue#L45

https://github.com/jellyfin/jellyfin-vue/blob/master/components/Item/Card/Card.vue#L81

https://github.com/jellyfin/jellyfin-vue/blob/master/components/Layout/AudioControls.vue#L129

https://github.com/jellyfin/jellyfin-vue/blob/master/components/Players/DraggableQueue.vue#L42

https://github.com/jellyfin/jellyfin-vue/blob/master/components/Players/TrackList.vue#L43

https://github.com/jellyfin/jellyfin-vue/blob/master/pages/item/_itemId/index.vue#L58

Comment (ferferga) [reply components/Buttons/LikeButton.vue:2]: @Camc314 This is a misstep from my rebase of #609. Can you add it to card, please?

If you go to light mode in the current state, the like button is not visible, so we need it for cards. No problems for the rest of the UI.

Comment (Camc314) [reply components/Buttons/LikeButton.vue]: I've added the prop back to the card

File: components/Buttons/__tests__/LikeButton.spec.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import { createLocalVue, mount } from '@vue/test-utils';
+import Vuetify from 'vuetify';
+import Vuex, { Store } from 'vuex';
+import LikeButton from '../LikeButton.vue';
+
+describe('Like button', () => {
+  const localVue = createLocalVue();
+  let vuetify: Vuetify;
+  let store: Store<unknown>;
+  let wrapper: any;
```
Comment (ferferga) [inline components/Buttons/__tests__/LikeButton.spec.ts:11]:
```suggestion
  let wrapper: Vue;
```

**Review by ferferga — APPROVED** [review]

**Pull request merged**